code/obj/artifacts/artifactprocs.dm

```diff
@@ -162,6 +162,9 @@
 	else
 		A.show_fx(src)
 	A.effect_activate(src)
+	for (var/mob/living/L in range(5, src))
+		if (ishuman(L) || isrobot(L))
```

Comment:
```suggestion
		if (L.mind)
```
this is probably what you mean and `L.mind.foo` is unsafe otherwise

Reply:
Not exactly, otherwise it will count intangibles, critters, etc